### DISCUSSION TITLE
How to change player hitpoints

### DISCUSSION THREAD

**Discussion by TheNotEasy** [efroemling/ballistica#350] [Q&A] [answered]
How to change player hitpoints? I cannot find the variable hitpoints in the ba.Player class

**Comment by Voxel25** [comment, marked as answer]
[PlayerSpaz](https://github.com/efroemling/ballistica/blob/master/assets/src/ba_data/python/bastd/actor/playerspaz.py) inherits [hitpoints](https://github.com/efroemling/ballistica/blob/master/assets/src/ba_data/python/bastd/actor/spaz.py#L172) from [Spaz](https://github.com/efroemling/ballistica/blob/master/assets/src/ba_data/python/bastd/actor/spaz.py)
It's stored in:
`player.actor.hitpoints`

**Reply by Dliwk** [reply]
Addition: `ba.Player` is only wrapper around `ba.Actor` which can be `ba.Spaz` (which has hitpoints).
